core/templates/dev/head/domain/exploration/SolutionObjectFactory.js

```diff
@@ -68,8 +68,7 @@ oppia.factory('SolutionObjectFactory', [
       var explanation = (
         $filter('convertToPlainText')(this.explanation));
       return (
-        '[' + solutionType + ' solution is ' + correctAnswer + '] ' +
-        explanation);
+        solutionType + ' solution is ' + correctAnswer + '. ' + explanation);
```

Comment:
> Since it is a list it has braces and quotation marks. I could create specific summaries based on interactions in the SolutionObjectFactory.getSummary function.

Nah, don't bother. I don't care about this too strongly; just do something simple that makes sense in the majority of cases (so, either have quotation marks for everything, or for nothing). We'll need to tidy up the finer display of this anyway at some point.